.travis.yml

```diff
@@ -13,6 +13,10 @@ before_script:
     - pip install -r requirements_test.txt
 script:
     - coverage run --source payu test_payu.py
+
+matrix:
+    allow_failures:
+        - python: "2.6"
```

Comment:
I think just remove it from the tests, rather than run-and-ignore.  We may be stuck with a 2.6 environment, but it's just becoming impossible to get support from 2.6 modules.